README.md

```diff
@@ -73,6 +73,11 @@ You can read more about it [here](https://www.typescriptlang.org/tsconfig#esModu
 
 * Your function is now fully instrumented
 
+## Connect Your Lumigo Account
+Set your Lumigo token as the `LUMIGO_TRACER_TOKEN` environment variable of your Lambda function; refer to the [Using AWS Lambda environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html) documentation for more information. Your Lumigo token is available in `Settings -> Tracing -> Manual tracing`, see the [Lumigo Tokens](https://docs.lumigo.io/docs/lumigo-tokens) documentation.
```

Comment:
Here you are directing users to set the `LUMIGO_TRACER_TOKEN` env var, but before you directed them to specify the value inside their js/ts code

Reply:
This PR removes the bit where users are directed to use the value inside their js/ts code, and directs them to use the `LUMIGO_TRACER_TOKEN` env var. The purpose of this PR is to basically have the readme page match the [documentation we have on the website](https://docs.lumigo.io/docs/nodejs).

Reply:
Yea got it, I read the PR the opposite way